contracts/staking/TokenStaking.sol

```diff
@@ -171,9 +176,13 @@ contract TokenStaking is Ownable, IStaking, Checkpoints {
         address indexed oldOwner,
         address indexed newOwner
     );
+    event GovernanceTransferred(
+        address indexed previousGovernance,
+        address indexed newGovernance
```

Comment:
Should we keep them indexed? It's something that is happening once in a lifetime, if ever.